composer.json

```diff
@@ -24,7 +24,7 @@
     "require-dev": {
         "atlas/testing": "~1.0",
         "pds/skeleton": "~1.0",
-        "phpunit/phpunit": "~7.0"
+        "phpunit/phpunit": "~7.0 || ~9.0"
```

Comment:
[Version 7 of PHPUnit is dead](https://phpunit.de/supported-versions.html), also [version 7 of PHP](https://www.php.net/supported-versions).